rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/db/OpenCloseTest.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Sme.UP S.p.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.smeup.rpgparser.db
+
+import com.smeup.rpgparser.AbstractTest
+import org.junit.Test
+import kotlin.test.assertEquals
+
+open class OpenCloseTest : AbstractTest() {
+
+    @Test
+    fun testT53_A01() {
+        val outputLines = outputOfDBPgm(
+            "db/T53_A01",
```

Comment:
Change the test name.
You are calling T53_A01 a program that is different from the MULANGT53 section A01 and this can generate confusion.

Reply:
Done